config/webpack-config.js

```diff
@@ -4,25 +4,34 @@
 
 'use strict';
 
-var BannerFooterPlugin = require( 'banner-footer-webpack-plugin' );
-var path = require( 'path' );
-var paths = require( '../config/environment' ).paths;
-var scriptsManifest = require( '../gulp/utils/scripts-manifest' );
-var webpack = require( 'webpack' );
+const BannerFooterPlugin = require( 'banner-footer-webpack-plugin' );
+const path = require( 'path' );
+const environment = require( '../config/environment' );
+const paths = environment.paths;
+const scriptsManifest = require( '../gulp/utils/scripts-manifest' );
+const webpack = require( 'webpack' );
 
 // Constants.
-var JS_ROUTES_PATH = '/js/routes';
-var COMMON_BUNDLE_NAME = 'common.js';
+const JS_ROUTES_PATH = '/js/routes';
+const COMMON_BUNDLE_NAME = 'common.js';
 
-var modernConf = {
+const modernConf = {
   cache: true,
   context: path.join( __dirname, '/../', paths.unprocessed, JS_ROUTES_PATH ),
   entry: scriptsManifest.getDirectoryMap( paths.unprocessed + JS_ROUTES_PATH ),
   module: {
     rules: [ {
       test: /\.js$/,
       use: [ {
-        loader: 'babel-loader?cacheDirectory=true'
+        loader: 'babel-loader?cacheDirectory=true',
+        options: {
+          presets: [ [ 'env', {
+            targets: {
+              browsers: environment.getSupportedBrowserList()
+            },
+            debug: true
```

Comment:
Could be, but since there's already non-optional output generated by the build process it seemed helpful to always display what browsers are being targeted. Would there be a reason to turn it off?